src/reducers/articleReducer.js

```diff
@@ -0,0 +1,36 @@
+import { GET_SINGLE_ARTICLE } from '../action-types';
+
+const defualtState = {
+  author: {
+    id: '',
+    bio: '',
+    image: '',
+    username: '',
+  },
+  id: '',
+  body: '',
+  claps: '',
+  comments: '',
+  description: '',
+  rating: '',
+  readingTime: '',
+  slug: '',
+  title: ''
+};
+
+const articlReducer = (state = defualtState, action) => {
```

Comment:
Is the name here deliberate? `articlReducer`

Reply:
No, it was a mistake, thanks